polly/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "polly"
+version = "0.0.1"
+authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
+
+[dependencies]
+libc = "0.1.12"
```

Comment:
Can we use a more recent version of libc? We already have a dependency on libc in Firecracker and that one is 0.2.64. Just specifying 0.2 should suffice. Generally, it is a good idea to have the same version of a dependency where this is possible.